src/kernel/thread/thread_sched_wait.c

```diff
@@ -47,17 +47,26 @@ static void sched_wait_timeout_handler(struct sys_timer *timer, void *data) {
 #define SCHED_WAIT_TIMER_INIT(_tmr, ...) \
 	timer_init_start(&_tmr, __VA_ARGS__)
 
+#define SCHED_WAIT_TIMER_ADD(_tmr) \
+		thread_wait_add(&(thread_self()->thread_wait), &_tmr)
```

Comment:
Here the second argument is `&_tmr`, but in the other branch of `#ifdef` it's `_tmr`, is it correct?